subprojects/problems/src/main/java/org/gradle/api/problems/ProblemBuilder.java

```diff
@@ -51,17 +40,52 @@
 @Incubating
 public interface ProblemBuilder {
 
+    /**
+     * The long description of this problem.
+     *
+     * @param details the details
+     *
+     * @return this
```

Comment:
🤔 I'm not sure the `@return` entries are required here, especially if `this` is the only comment.
I would rather change it to something more descriptive, or delete it.

Reply:
E.g.,
```suggestion
     * @return the same instance the method was called on
```

Reply:
I have to disagree here. We use `* @return this` regularly in the Gradle javadoc.